### PR TITLE
Retrieve commands history from API

### PR DESCRIPTION
**Pull request by Franceq34**
- related https://github.com/jhipster/jhipster-lite/issues/1015

### CODE REVIEW

File: src/main/webapp/app/common/secondary/ProjectHistoryRepository.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { RestHistory, toHistory } from '@/common/secondary/RestHistory';
+import { History } from '@/common/domain/History';
+import { ProjectHistoryService } from '@/common/domain/ProjectHistoryService';
+import { AxiosHttp } from '@/http/AxiosHttp';
+
+export default class ProjectHistoryRepository implements ProjectHistoryService {
+  constructor(private axiosHttp: AxiosHttp) {}
+
+  async get(folder: Folder): Promise<History> {
+    return this.axiosHttp.get<RestHistory>(`api/projects/history?folder=${folder}`).then(response => toHistory(response.data));
```
Comment (seraphinandrieux) [inline src/main/webapp/app/common/secondary/ProjectHistoryRepository.ts:11]: What do you think about request param folder into params attribute into get config ? Something like : `this.axiosHttp.get<RestHistory>('api/projects/history',{
params:{
folder}})`

Comment (seraphinandrieux) [inline src/main/webapp/app/common/secondary/ProjectHistoryRepository.ts:11]: What do you think about request param folder into params attribute into get config ? Something like : `this.axiosHttp.get<RestHistory>('api/projects/history',{
params:{
folder}})`

File: src/test/javascript/spec/common/secondary/ProjectHistoryRepository.spec.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+describe('ProjectRepository', () => {
+  it('should get project history', async () => {
+    const axiosHttpStub = stubAxiosHttp();
+    axiosHttpStub.get.resolves({ data: createRestHistory() });
+    const projectHistoryRepository = new ProjectHistoryRepository(axiosHttpStub);
+
+    const history: History = await projectHistoryRepository.get('folder/path');
+
+    const [uri] = axiosHttpStub.get.getCall(0).args;
```
Comment (seraphinandrieux) [inline src/test/javascript/spec/common/secondary/ProjectHistoryRepository.spec.ts:15]: If you use config params, you could separate uri from params and do a separate assertion.
`const [uri,params] = axiosHttpStub.get.getCall(0).args;
expect(uri).toBe('api/projects/history');
expect(params).toBe({folder:'folder/path'});

`

Comment (seraphinandrieux) [inline src/test/javascript/spec/common/secondary/ProjectHistoryRepository.spec.ts:15]: If you use config params, you could separate uri from params and do a separate assertion.
`const [uri,params] = axiosHttpStub.get.getCall(0).args;
expect(uri).toBe('api/projects/history');
expect(params).toBe({folder:'folder/path'});

`

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**